js1/funkce-obory/cvdoma/kalkulacka.md

```diff
@@ -9,13 +10,10 @@ Napište funkci `calculate` se třemi parametry `number1`, `operation` a `number
 
 Příklad použití
 
-```jscon
-> calculate(2, '+', 3)
-5
-> calculate(3, '*', 7)
-21
-> calculate(10, '/', 4)
-2.5
+```js
+document.innerHTML += '2 + 3 = ' + calculate(2, '+', 3) + '<br>' // vypíše výsledek 5
+document.innerHTML += '3 * 7 = ' + calculate(3, '*', 7) + '<br>' // vypíše výsledek 21
+document.innerHTML += '10 / 4 = ' + calculate(10, '+', 4) + '<br>' // vypíše výsledek 2.5
```

Comment:
Když už znají interpolaci řetězců, nebyla by lepší ta (ať se tolik nemíchá všelijaké sčítání)?

Reply:
Uvažoval jsem nad tím, ale říkal jsem si, zda to ještě není příliš komplikované, volat v interpolaci přímo funkci. Myslím, že jsme zatím vždy ukazovali interpolaci jen s proměnnou.
Případně by bylo možné vždy výsledek uložit do proměnné a tu pak vypsat v interpolaci, ale tam se zase ztrácí to, že je volání funkce hned v místě, kde se vkládá výsledek.
Připadá mi, že jsou to tři špatná řešení 🤷🏻‍♂️

Reply:
Hmm. Dobrá poznámka. Asi to tada klidně nech takto. Mně se jen nezdálo, že na jednom řádku mají tři různá pluska a každé znamená něco jiného. :D

Reply:
Interpolaci s volálním funkce už jsme ukazovali hned na hodině, kde se probírala interpolace, takže já jsem tady taky pro interpolaci.

Reply:
Upravil jsem to na interpolaci.